HelpSource/Classes/TextView.schelp

```diff
@@ -269,7 +269,7 @@ t = TextView(w.asView,Rect(10,10, 500,200))
 )
 
 // Using the Window you just created, try these in succession, and test how the text view responds
-t.mouseUpAction_{|it, x, y, modifiers, buttonNumber, clickCount, pos| [pos].postln};
+t.mouseUpAction_{|it, x, y, modifiers, buttonNumber| [pos].postln};
```

Comment:
there is no `pos` argument defined. Maybe you forgot to add it?

Reply:
oops, thanks for catching that.  There is no pos arg passed anymore, something else should be printed instead.